library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -97,7 +98,10 @@ void setupLabel() {
         label.setText(parseLabel(labelText));
         label.setTextColor(Color.parseColor(LABEL_TEXT_COLOR));
         label.setTextSize(TypedValue.COMPLEX_UNIT_SP, LABEL_FONT_SIZE);
-        label.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/josefin_sans.ttf"));
+        if(LABEL_FONT!=null)
+            label.setTypeface(Typeface.createFromAsset(context.getAssets(), LABEL_FONT));
+        else
+            label.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/josefin_sans.ttf"));
```

Comment:
Please remove the `else` case, default font to be used should be the system font